[Chronicle-246] Google Chronicle Backstory Fixes & Enhancements (#12845)
demisto/content: Packs/GoogleChronicleBackstory/ReleaseNotes/2_0_1.md

#### Classifiers
##### New: Chronicle - Incoming Mapper
- Maps incoming Chronicle incident fields. (Available from Cortex XSOAR 6.0.0).
##### New: Google Chronicle Backstory
-  (Available from Cortex XSOAR 5.0.0).

#### Integrations
##### Chronicle

> - Added simple_backoff_rules dictionary to track retry attempts for 429 and 500 error and if 400 or 404 error occurs for detection of any rule , it will be skipped. Added 'gcb-list-rules' command.

Reviewer: ```suggestion
- Added simple_backoff_rules dictionary to track retry attempts for 429 and 500 error and if 400 or 404 error occurs for detection of any rule.
- Added the ***gcb-list-rules*** command.
```

##### New: list_detections_hr.md
-  (Available from Cortex XSOAR 5.0.0).

#### Scripts
##### ChronicleAssetEventsForHostnameWidgetScript

> - Updated the Docker image to demisto/python3:3.9.1.15759.

Reviewer: ```suggestion
- Updated the Docker image to demisto/python3:3.9.5.20070.
```

please change this for all the scripts you updated the docker
